src/api/Jalasoft.TeamUp.Projects.API/Controllers/GetHealth.cs

```diff
@@ -16,21 +16,20 @@
 
     public class GetHealth
     {
-        private readonly IHealthsService projectsService;
+        private readonly IHealthsService healthsService;
 
-        public GetHealth(IHealthsService projectsService)
+        public GetHealth(IHealthsService healthsService)
```

Comment:
I know it's not your code but what's the convention for the health endpoint here? Can you check and align that for me please? It's different from resumes project atm

Reply:
I could read something about that, and health is used without **s** :D